cluster-provision/k8s/1.20/provision.sh

```diff
@@ -0,0 +1,327 @@
+    device-mapper-persistent-data \
+    lvm2
+
+# Add Docker repository.
+dnf config-manager --add-repo=https://download.docker.com/linux/centos/docker-ce.repo
+
+# Enable the nightly version to get a new enough runc version which includes containerd
+# with a new enough runc version to avoid https://github.com/kubernetes/kubernetes/issues/95296#issuecomment-714419461
+# which manifests in /dev/* permission denied flakes on cpu manager enabled nodes
+dnf config-manager --set-enabled docker-ce-nightly
+
+# Install package(s) that trigger the enablement of the container-tools yum module
+dnf -y install container-selinux
+
+# Disable the container-tools module, as it forces containerd.io to an ancient version,
+#   which in turn forces docker-ce to an older version, making it incompatible with docker-ce-cli...
+dnf -y module disable container-tools
+
+# Install iptables needed for docker-ce
+dnf install -y iptables
+
+# Install Docker CE.
+dnf install -y docker-ce --nobest
+
+# Create /etc/docker directory.
+mkdir /etc/docker
+
+# Setup docker daemon
+cat << EOF > /etc/docker/daemon.json
+{
+  "insecure-registries" : ["registry:5000"],
+  "log-driver": "json-file",
+  "exec-opts": ["native.cgroupdriver=systemd"],
+  "ipv6": true,
+  "fixed-cidr-v6": "2001:db9:1::/64",
+  "selinux-enabled": true
+}
+EOF
+
+mkdir -p /etc/systemd/system/docker.service.d
+
+# Restart Docker
+systemctl daemon-reload
+systemctl restart docker
+
+#TODO: el8 repo
+# Add Kubernetes repository.
+cat <<EOF >/etc/yum.repos.d/kubernetes.repo
+[kubernetes]
+name=Kubernetes
+baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64
+enabled=1
+gpgcheck=1
+repo_gpgcheck=0
+gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+EOF
+
+# Install Kubernetes packages.
+dnf install --skip-broken --nobest --nogpgcheck --disableexcludes=kubernetes -y \
+    kubeadm-${version} \
+    kubelet-${version} \
+    kubectl-${version} \
+    kubernetes-cni
+
+# TODO use config file! this is deprecated
+cat <<EOT >/etc/sysconfig/kubelet
+KUBELET_EXTRA_ARGS=--cgroup-driver=systemd --runtime-cgroups=/systemd/system.slice --kubelet-cgroups=/systemd/system.slice --feature-gates="BlockVolume=true,CSIBlockVolume=true,VolumeSnapshotDataSource=true,IPv6DualStack=true"
+EOT
+
+systemctl daemon-reload
+
+systemctl enable docker && systemctl start docker
+systemctl enable kubelet && systemctl start kubelet
+
+# Needed for kubernetes service routing and dns
+# https://github.com/kubernetes/kubernetes/issues/33798#issuecomment-250962627
+modprobe bridge
+modprobe br_netfilter
+cat <<EOF >  /etc/sysctl.d/k8s.conf
+net.bridge.bridge-nf-call-iptables = 1
+net.ipv4.ip_forward = 1
+net.ipv6.conf.all.disable_ipv6 = 0
+net.ipv6.conf.all.forwarding = 1
+net.bridge.bridge-nf-call-ip6tables = 1
+EOF
+sysctl --system
+
+echo bridge >> /etc/modules
+echo br_netfilter >> /etc/modules
+
+# Bump NetworkManager to 1.22.8
+NM_VERSION=1.22.8
```

Comment:
THis is the issue:

```
No match for argument: NetworkManager-1.22.8
Error: Unable to find a match: NetworkManager-1.22.8
```

you will have to bump it to `1.30.0`.

Reply:
Can we make it 1.26 ? if not kubernetes-nmstate has to downgrade to it.

Reply:
What do you mean with downgrade? The suggested 1.30 is newer and I can't see a 1.26 in the centos  stream: http://mirror.23media.com/centos/8-stream/BaseOS/x86_64/os/Packages/. Only 1.30.

Reply:
ok, let's just remove this check, if needed at kubernetes-nmstate CI we can ping to 1.26 or the like using copr.

Reply:
Currently check-provision-1.19 lane is failing and blocking PR's due to that update.
I think we should drop this NM update or add different repository to get the pinned version,
because we bound to what is provided with Cetos Stream

Reply:
I am not yet removing the check, but this bump PR #497 solves that.